public/app/features/alerting/unified/RuleList.tsx

```diff
@@ -32,6 +33,8 @@ const VIEWS = {
   state: RuleListStateView,
 };
 
+const onExport = () => window.open(`/api/v1/provisioning/alert-rules/export?download=true`);
```

Comment:
Love this feature. Unfortunately, this doesn't work when grafana is served from a sub path, such as https://somedomain.com/grafana

Reply:
thank you for spotting this out! @MiggL

Reply:
I've a PR up here https://github.com/grafana/grafana/pull/64136